README.md

```diff
@@ -0,0 +1,98 @@
+# Joy2Twist
+
+Dockerized ROS node allowing to control ROS powered mobile robots with Logitech F710 gamepad. Joy2Twist node is converting `sensor_msgs/Joy` message to `geometry_msgs/Twist` in order to provide velocity commands for mobile robot.
```

Comment:
Are we keeping it to work with only `Logitech F710`?

Reply:
Ta paczka ma demo, które odpala pod spodem drivera logitech f710. Dodam po prostu notkę, że jest zgodność z każdym kontolerem, który publikuje message Joy'a

Reply:
:heavy_check_mark: